pkg/controller/config.go

```diff
@@ -180,7 +180,6 @@ func NewConfig() *ControllerConfig {
 		AciVmmDomainType:   "Kubernetes",
```

Comment:
There should be an info log statement somewhere around here which states what value is actually picked up and being used.

Reply:
Added in controller.go